selftests/unit/test_test.py

```diff
@@ -111,13 +111,6 @@ def test_no_data_dir(self):
         tst = self._get_fake_filename_test(above_limit_name)
         self.assertFalse(tst.get_data('', 'file', False))
 
-    def test_all_dirs_exists_no_hang(self):
```

Comment:
IMO, this test is not relevant on the nrunner, because this check is part of [legacy code](https://github.com/avocado-framework/avocado/blob/master/avocado/core/test.py#L288).